apps/openmw/mwworld/weather.cpp

```diff
@@ -289,44 +294,54 @@ WeatherManager::WeatherManager(MWRender::RenderingManager* rendering, MWWorld::F
 
     //Weather
     Weather clear;
+    clear.mObstructsCelestialBodies = false;
```

Comment:
I did some experiments in vanilla MW to see if this is really hardcoded. I believe it's the
<pre>
Glare View=1
</pre>
weather setting.